src/apiendpoint.ts

```diff
@@ -8,14 +8,10 @@ export namespace Apiendpoint {
   export let STACK_API_URL: string = '';
   export let OSIO_ROUTE_URL: string = '';
   export const STACK_REPORT_URL: string =
-    'https://stack-analytics-report.prod-preview.openshift.io/';
+    'https://stack-analytics-report.openshift.io/';
```

Comment:
@dvandra i would suggest not to do this change for now as Stack Report UI Changes are not in prod.
@arajkumar wdys?

Reply:
Revert this as well, raise another PR to put back staging.